ui/less/containers.less

```diff
@@ -256,6 +256,13 @@
     color: white;
     cursor: pointer;
   }
+
```

Comment:
Now that more buttons are available in the context menu (and as they will have different widths due to the length of their names) I think we could also add `width: 100%` to the `button` selector in `.shaka-context-menu`, to maintain the sizes of the selector the same.

Reply:
Done!